<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmation_page</class>
 <widget class="QWidget" name="confirmation_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>448</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="0">
        <widget class="QCheckBox" name="download_llm_checkbox">
         <property name="text">
          <string>Download LLM</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="compile_with_pyinstaller_checkbox">
         <property name="text">
          <string>Compile with Pyinstaller</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="download_controlnet_checkbox">
         <property name="text">
          <string>Download Controlnet models</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="download_sd_checkbox">
         <property name="text">
          <string>Download Stable Diffusion models</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="required_libraries_groupbox">
         <property name="title">
          <string>Install required libraries (linux)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QScrollArea" name="required_libraries">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>404</width>
               <height>128</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_4"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="download_ai_runner_checkbox">
         <property name="text">
          <string>Download AI Runner</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="download_stt_checkbox">
         <property name="text">
          <string>Download Speech to Text models</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirm your settings and click next to install AI Runner</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="download_tts_checkbox">
         <property name="text">
          <string>Download Text to Speech models</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>download_ai_runner_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_ai_runner(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_sd_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_sd(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_controlnet_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_controlnet(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_llm_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_llm(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_tts_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_tts(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_stt_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_download_stt(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>required_libraries_groupbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_all_required(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>compile_with_pyinstaller_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmation_page</receiver>
   <slot>toggle_compile_with_pyinstaller(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>agreement_clicked(bool)</slot>
  <slot>toggle_download_ai_runner(bool)</slot>
  <slot>toggle_download_sd(bool)</slot>
  <slot>toggle_download_controlnet(bool)</slot>
  <slot>toggle_download_llm(bool)</slot>
  <slot>toggle_download_tts(bool)</slot>
  <slot>toggle_download_stt(bool)</slot>
  <slot>toggle_all_required(bool)</slot>
  <slot>toggle_emoji_fonts(bool)</slot>
  <slot>toggle_compile_with_pyinstaller(bool)</slot>
 </slots>
</ui>
